.github/workflows/check.yaml

```
@@ -0,0 +1,35 @@
+name: check
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        emacs_version:
+          - 28.2
+          - 27.1
+          - 26.1
+          - 26.2
+          - 26.3
+        ignore_warnings:
+          - false
+        include:
+          - emacs_version: snapshot
+            ignore_warnings: false
```

Yay! I think this would have been the correct change:
```suggestion
          - snapshot
        ignore_warnings:
          - false
```

---

Nice!